resources/scripts/composer/post-install.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+# Symlink the git pre-push hook to its destination.
+if [ ! -h ".git/hooks/pre-push" ] ; then
+  ln -s "../../vendor/pfrenssen/phpcs-pre-push/pre-push" ".git/hooks/pre-push"
```

Comment:
Good suggestion. I solved it by removing the symlink if it is present, and then creating the new symlink.